src/Symfony/Component/Security/Http/Tests/Authentication/AuthenticatorManagerTest.php

```diff
@@ -365,28 +450,32 @@ public function log($level, $message, array $context = []): void
             }
         };
 
-        $manager = $this->createManager([$authenticator], 'main', true, [], $logger);
+        $manager = $this->createManager([$authenticator], 'main', \true, [], $logger, hideUserNotFoundExceptions: \true);
         $response = $manager->authenticateRequest($this->request);
         $this->assertSame($this->response, $response);
         $this->assertStringContainsString($authenticator::class, $logger->logContexts[0]['authenticator']);
     }
 
-    private function createAuthenticator(?bool $supports = true)
+    private function createAuthenticator(?bool $supports = \true)
     {
         $authenticator = $this->createMock(TestInteractiveAuthenticator::class);
         $authenticator->expects($this->any())->method('supports')->willReturn($supports);
 
         return $authenticator;
     }
 
-    private static function createDummySupportsAuthenticator(?bool $supports = true)
+    private static function createDummySupportsAuthenticator(?bool $supports = \true)
     {
         return new DummySupportsAuthenticator($supports);
     }
 
-    private function createManager($authenticators, $firewallName = 'main', $eraseCredentials = true, array $requiredBadges = [], ?LoggerInterface $logger = null)
+    private function createManager($authenticators, $firewallName = 'main', $eraseCredentials = \true, array $requiredBadges = [], ?LoggerInterface $logger = null, string $exposeSecurityErrors = AuthenticatorManager::ACCOUNT_STATUS, ?bool $hideUserNotFoundExceptions = null)
     {
-        return new AuthenticatorManager($authenticators, $this->tokenStorage, $this->eventDispatcher, $firewallName, $logger, $eraseCredentials, true, $requiredBadges);
+        if (true === $hideUserNotFoundExceptions) {
+            $this->expectUserDeprecationMessage('Since symfony/security-http 7.2: Setting the "hideUserNotFoundExceptions" parameter is deprecated, use "exposeSecurityErrors" instead.');
+        }
+
+        return new AuthenticatorManager($authenticators, $this->tokenStorage, $this->eventDispatcher, $firewallName, $logger, $eraseCredentials, $hideUserNotFoundExceptions, $requiredBadges, exposeSecurityErrors: $exposeSecurityErrors);
```

Comment:
Don't use named arguments. Our BC policy does not guarantee BC for argument names. If we write tests using named arguments, it will hide cases where we break BC for positional arguments as tests will keep working without any change, which is a bad outcome.